less/index.less

```diff
@@ -54,6 +54,104 @@ table {
 }
 
 html, body {
-    height: 100%;
+	height: 100%;
 }
 
+@variable-color: #061B47;
+@color2: #042F50;
+@color3: #0A0A59;
+
+body {
+	background: #9CBEFC;
+h1 {
+	font-size: 30px;
+	color: @variable-color;
+}
+h2 {
+	font-size: 24px;
+	color: @variable-color;
+}
+h3 {
+	font-size: 20px;
+	color: @color3;
+}
+
+header {
+    width: 100%;
+	display: flex;
+	@media (max-width: 500px) {
+			justify-content: center;
+			align-content: center;
+		}
+}
+
+header nav {
+    margin-top: 40px; 
+    width: 610px;
+    justify-content: space-between;
+	display: flex;
+	@media (max-width: 500px) {
+		width: 100%;
+		margin: 0;
+	}
+}
+
+header nav a {
+    text-decoration: none;
+	color: @variable-color;
+	@media (max-width: 500px) {
+		width: 100%;
+		display: flex;
+		text-align: center;
+		padding: 20px;
+		font-size: 20px;
+	}
+}
+
+header nav a:hover {
+	color: #D9C6D7;
+}
+.mixin1 {
+	display: flex;
+	flex-direction: column;
+	padding: 20px;
+	align-items: flex-start;
+}
+.mixin2 {
+	display: flex;
+	flex-direction: column;
+	margin-top: 20px;
+	align-items: flex-start;
+}
+.Rtitle {
+	.mixin1();
+}
+.Intro {
+	.mixin1();
+	color: @variable-color;
+	.ip {
+	margin-top: 20px;
+	}
+}
+.Skills {
+	.mixin1();
+	color: @variable-color;
+	.sp {
+	margin-top: 20px;
+	}
+}
+.WH {
+	.mixin1();
+	.lw {
```

Comment:
try to use more explicit class names.  also, when you define mixins, it's helpful to add the parenthesis so you can distinguish it from a class.